services/src/main/java/io/druid/cli/CliPeon.java

```diff
@@ -157,6 +156,7 @@ public void configure(Binder binder)
             binder.bind(DataSegmentArchiver.class).to(OmniDataSegmentArchiver.class).in(LazySingleton.class);
 
             binder.bind(ExecutorLifecycle.class).in(ManageLifecycle.class);
```

Comment:
The LifecycleModule always snags me up in when things are created vs just registered. Is this line still needed?